lta/rest_server.py

```diff
@@ -766,11 +782,18 @@ def start(debug: bool = False) -> RestServer:
         else:
             logging.info(f"{name} = NOT SPECIFIED")
 
-    args = RestHandlerSetup({  # type: ignore
-        "auth": {
+    if config["CI_TEST"] == "TRUE":
+        auth = {
+            "secret": "secret",
+        }
+    else:
+        auth = {
             "audience": config["LTA_AUTH_AUDIENCE"],
-            "openid_url": config["LTA_AUTH_OPENID_URL"],
-        },
+            "openid_url": config["LTA_AUTH_OPENID_URL"]
+        }
+
+    args = RestHandlerSetup({  # type: ignore
+        "auth": auth,
```

Comment:
This is really elegant; much nicer than the hacky global scope lta_auth swap. Thank you for this.